tikv-configuration-file.md

```diff
@@ -2501,3 +2501,36 @@ Configuration items related to [Load Base Split](/configure-load-base-split.md).
 
 + Specifies the amount of data sampled by Heap Profiling each time, rounding up to the nearest power of 2.
 + Default value: `512KiB`
+
+## in-memory-engine <span class="version-mark">New in v8.5.0</span>
+
+TiKV MVCC in-memory engine configuration items related to the storage layer.
+
+### `enable` <span class="version-mark">New in v8.5.0</span>
+
+> **Note:**
+>
+> This configuration item cannot be queried via SQL statements but can be configured in the configuration file.
+
++ Whether to enable the in-memory engine to accelerate multi-version queries. For more information on the in-memory engine, refer to [TiKV MVCC In-Memory Engine](/tikv-in-memory-engine.md)
++ Default value: false (the in-memory engine is disabled)
+
+### `capacity` <span class="version-mark">New in v8.5.0</span>
+
+> **Note:**
+>
+> + When the in-memory engine is enabled, `block-cache.capacity` will automatically decrease by 10%.
+> + When manually configuring `capacity`, `block-cache.capacity` will not automatically decrease, and you need to manually adjust the value to avoid OOM.
```

Comment:
```suggestion
> + When you manually configure `capacity`, `block-cache.capacity` will not automatically decrease. In this case, you need to manually adjust the value to avoid OOM.
```